backend/corpora/common/corpora_orm.py

```diff
@@ -299,3 +303,6 @@ class DbDatasetContributor(Base):
     dataset_id = Column(ForeignKey("dataset.id"), nullable=False)
     created_at = Column(DateTime, nullable=False, server_default=DEFAULT_DATETIME)
     updated_at = Column(DateTime, nullable=False, server_default=DEFAULT_DATETIME)
+
+    contributor = relationship("DbContributor")
```

Comment:
You'll have to back_populate here once the relationship is added to DbContributors.